cpu/stm32/cpu_init.c

```diff
@@ -161,6 +161,7 @@ static void _gpio_init_ain(void)
  * This very teniously avoids optimization, even optimized it's better than
  * nothing but periodic review should establish that it doesn't get optimized.
  */
+#if defined(STM32_OPTION_BYTES)
```

Comment:
```suggestion
MAYBE_UNUSED
```

Reply:
Saw this comment too late, see https://github.com/RIOT-OS/RIOT/pull/18870#discussion_r1530308944